_breadcrumbs.scss

```diff
@@ -0,0 +1,116 @@
+// Breadcrumbs
+// ===
+
+
+// Configurable Variables
+// ---
+
+$breadcrumbs__divider-content: '>' !default; // TO DO: include stencil icon
+$breadcrumbs__show-more-content: '...' !default; // TO DO: include stencil icon
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/hide-text';
+@import 'spline/dist/utilities/icon-font-replace';
+@import 'level/level';
+
+
+// Breadcrumbs
+// ---
+//
+
+.c-breadcrumbs {
+    display: flex;
+    overflow: auto;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+
+    -webkit-overflow-scrolling: touch;
+}
+
+
+// Breadcrumbs item
+// ---
+//
+// 1. Vertical align text center
+// 2. Hide breacrumb item with hidden attribute
+
+.c-breadcrumbs__item {
+    display: flex; // 1
+    align-items: center; // 1
+    flex: 0 0 auto;
+
+    &[hidden] {
+        display: none; // 2
+    }
+
+    // Active style
+    // ---
+    //
+
+    &.c--active {
+
+        .c-breadcrumbs__link {
+            font-weight: bold;
+        }
+    }
+
+    // Breadcrumbs divider
+    // ---
+    //
+    // 1. Divider content
+
+    + .c-breadcrumbs__item {
+        &::before {
+            content: $breadcrumbs__divider-content; // 1
+
+            display: inline-block;
+        }
+    }
+}
+
+
+// Breadcrumbs link
+// ---
+//
+
+.c-breadcrumbs__link {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
+    padding: $unit;
+
+    text-decoration: none;
+
+    // Breadcrumbs show more link
+    // ---
+    //
+    // 1. Add space between icon and carets
+    // 2. Remove padding so wont display text
+    // 3. Align icon to breadcrumb link
+    // 4. 'Show more' content
+
+    &.c--show-more {
+        @include icon-font-replace();
+
+        width: $tap-size;
+        height: $tap-size;
+        margin: 0 $unit; // 1
```

Comment:
Could the margin between items here throw off the calculation in the UI script? The calculation there adds up the width the of the items, which won’t account for the margin here.